src/v3/src/transformer/terminal/transformUnhandledErrors.test.ts

```diff
@@ -30,7 +30,7 @@ describe('Unhandled Error Transformer Tests', () => {
   });
 
   it('should add Infobox with unexpected error message when error is not provided', () => {
-    const formBag = transformUnhandledErrors(widgetProps);
+    const formBag = transformUnhandledErrors(widgetProps, undefined);
```

Comment:
Is explicitly passing `undefined` necessary?

Reply:
it was necessary since i changed the error parameter to be required, but I changed it back to optional so this isn't necessary anymore